Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -406,6 +407,46 @@ public function testWithRedisHandler()
         $this->assertEquals('monolog_redis_test', $config['handlers']['redis']['redis']['key_name']);
     }
 
+    /**
+     * @group legacy
+     */
+    public function testConsoleFormatterOptionsRename()
```

Comment:
please add another legacy test involving 2 configs, one migrated and one not migrated for the same handler.

Reply:
or maybe just another handler in this test

Reply:
Test updated